jupyter_client/manager.py

```diff
@@ -55,39 +58,45 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._shutdown_status = _ShutdownStatus.Unset
 
-    _created_context = Bool(False)
+    _created_context: Bool = Bool(False)
 
     # The PyZMQ Context to use for communication with the kernel.
-    context = Instance(zmq.Context)
-    def _context_default(self):
+    context: Instance = Instance(zmq.Context)
+    def _context_default(self) -> zmq.Context:
```

Comment:
I was surprised not to see the `defaults` decorator here:
```python
    @defaults('context')
```
Am I missing something?